luigi/format.py

```diff
@@ -95,6 +96,12 @@ def create_subprocess(self, command):
         http://www.chiark.greenend.org.uk/ucgi/~cjwatson/blosxom/2009-07-02-python-sigpipe.html
```

Comment:
Should this be updated as well?

Reply:
I'm unfortunatley not that familiar with *nix so I can't really comment on whether or not this is still relevant.